docs/src/pages/[platform]/components/message/react.mdx

```diff
@@ -0,0 +1,228 @@
+import { Fragment } from '@/components/Fragment'; 
+import { Message, View, Link } from '@aws-amplify/ui-react';
+import { MessageDemo } from './demo';
+import { ComponentStyleDisplay } from '@/components/ComponentStyleDisplay';
+import ThemeExample from '@/components/ThemeExample.mdx';
+import { Example, ExampleCode } from '@/components/Example';
+import {
+  BasicUsage,
+  BasicWithDismiss,
+  HideIcon,
+  IconProviderExample,
+  MessageColorThemes,
+  MessageVariations,
+  WithCustomizedDismiss,
+  ComposableMessage,
+  MessageGlobalStyling,
+  MessageStyleProps,
+  MessageAsAlert,
+  MessageThemeExample
+} from './examples';
+
+## Demo
+
+<MessageDemo />
+
+## Usage
+
+Message can be customized via props as a basic component, or it can be used as a <a href="#composable">composable component</a> which provides more control over layout and functionality.
```

Comment:
super nit: prefer the markdown link
```suggestion
Message can be customized via props as a basic component, or it can be used as a [composable component](#composable) which provides more control over layout and functionality.
```

Reply:
fixed in latest, thanks!